File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = false;
        int len = s.length();

        if(len == 4 || len == 6){
            for(int i = 0; i < len; i++) {
                if(s.charAt(i) > 57) return answer;
            }
            answer = true;
        }else {
            return answer;
        }

        
        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

bool solution(const char* s) {
    
    if(strlen(s) != 4 && strlen(s) != 6){
        return false;
    }
    
    for(int i = 0; s[i] != '\0'; i++){
        if(s[i] >= 'a' && s[i] <= 'z'){
            return false;
        }else if(s[i] >= 'A' && s[i] <= 'Z'){
            return false;
        }
    }
    
    
    return true;
}